pynestml/codegeneration/nest_code_generator.py

```diff
@@ -384,6 +384,8 @@ def _get_synapse_model_namespace(self, synapse: ASTSynapse) -> Dict:
         namespace["printer"] = self._unitless_nest_gsl_printer
         namespace["assignments"] = NestAssignmentsHelper()
         namespace["names"] = self._nest_reference_converter
+        namespace["nest_reference_converter"] = self._nest_reference_converter
+        namespace["types_printer"] = self._types_printer
```

Comment:
These are already part of the NEST printer passed to the templates. Is there a need to pass them separately again?